admin_guide/managing_pods.adoc

```diff
@@ -146,16 +146,17 @@ endif::openshift-enterprise,openshift-origin[]
 
 [[admin-guide-controlling-egress-traffic]]
 == Controlling Egress Traffic
-As an {product-title} cluster administrator, you can control egress traffic
-using either a xref:admin-guide-deploying-an-egress-router-service[router] or
-xref:admin-guide-limit-pod-access-egress[firewall] methods.
+As an {product-title} cluster administrator, you can control egress traffic in two different ways:
+
+* Using an egress xref:admin-guide-deploying-an-egress-router-service[router], so that an external source treats traffic as though it were coming from a known source.
+* Using an egress xref:admin-guide-limit-pod-access-egress[firewall], where a cluster administrator can enforce acceptable (outbound) traffic policies, so that specific endpoints or IP ranges (subnets) are the only acceptable targets for the dynamic endpoints (pods within {product-title}) to talk to.
```

Comment:
`Using an egress xref:admin-guide-limit-pod-access-egress[firewall], where a cluster administrator` does not read right. Should this say `Using an egress xref:admin-guide-limit-pod-access-egress[firewall], allows a cluster administrator`?